src/main/java/com/ibm/devops/connect/CRPipeline/UploadJUnitTestResult.java

```diff
@@ -169,7 +169,7 @@ public FileUploader(Map<String, String> properties, TaskListener listener, Strin
             HttpEntity entity = MultipartEntityBuilder
                 .create()
                 .addTextBody("payload", payload.toString())
-                .addBinaryBody("testArtifact", new File(f, filePath), ContentType.create("application/octet-stream"), "filename")
+                .addBinaryBody("file", new File(f, filePath), ContentType.create("application/octet-stream"), "filename")
```

Comment:
why is this changing? did it change on the api side?

Reply:
Since we're sending to a new URL this was the time to make it more generic, as the prior renaming.